libs/components/src/lib/Accordion/index.module.scss

```diff
@@ -0,0 +1,88 @@
+@import "../../../../../libs/theme";
+
+.container {
+  width: 100%;
+  display: flex;
```

Comment:
we have a mixin for flex (flexbox).
check this file:
/libs/theme/_mixins.scss

Reply:
added @include flexbox to every place that is needed in scss file